docs/providers/documentation/auth0-provider.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Auth0"
+sidebarTitle: "Auth0 Provider"
+description: "Auth0 provider allows integration with Auth0 for user authentication and authorization."
```

Comment:
This is actually inaccurate. Can you please adjust?

Reply:
Yep, will  update it